### ISSUE TITLE
Are we due for another beta release since 1.13.35.2 ?

### ISSUE THREAD

**Issue by centminmod** [apache/incubator-pagespeed-ngx#1635]
There's been various commits to master branch since 1.13.35.2 stable release https://github.com/apache/incubator-pagespeed-ngx/compare/v1.13.35.2-stable...master. Are we due for a beta release soon ?

**Comment by oschaaf** [comment]
Yes, changes have landed that are release-worthy IMHO; I've been trying to find some time to prepare a new RC because I wanted to do this earlier, but I have been snowed under a bit. I will update this thread when there is news.

**Comment by centminmod** [comment]
cheers @oschaaf :)

**Comment by tobsch** [comment]
any news @oschaaf? :)

**Comment by oschaaf** [comment]
I've worked on producing a release candidate, I'll shoot for getting that review by the incubator PMC this week. I'm not sure how long we'll be going back and forth, but we're sailing!

**Comment by Fierralin** [comment]
I cannot build it following the wiki~ Could someone help me? thx

**Comment by oschaaf** [comment]
@Fierralin could you paste the commands you ran? (and their outputs)?

**Comment by IvanRF** [comment]
@oschaaf any news? 😃 
I saw [v1.14.36.1](https://github.com/apache/incubator-pagespeed-mod/releases/tag/v1.14.36.1) was released on the Apache module.

**Comment by oschaaf** [comment]
There currently a VOTE thread in progress for release candidate number 4 on releasing `Apache PageSpeed (incubating) 1.14.26.1`. Hopefully the VOTE will pass, and then we can ship :rocket:!
